packages/nextjs/hooks/scaffold-eth/useTransactor.tsx

```diff
@@ -0,0 +1,108 @@
+import { TransactionRequest, TransactionResponse, TransactionReceipt } from "@ethersproject/abstract-provider";
+import { SendTransactionResult } from "@wagmi/core";
+import { BigNumber, ethers, Signer } from "ethers";
+import { Deferrable } from "ethers/lib/utils";
+import { useSigner } from "wagmi";
+import toast from "react-hot-toast";
+import { getParsedEthersError } from "~~/components/scaffold-eth/Contract/utilsContract";
+import { toast as customToast } from "~~/components/scaffold-eth/index";
+
+const DEBUG = true;
```

Comment:
I know this is used a lot on SE-1, but let's not do it here for now.

We can use logs for errors (BTW, we should use `console.error()` instead of `console.log()`)